### PR TITLE
[WIP] add job form

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Refers/Fixes

REFS: #3382 #3586 #3518 #3585
###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/dashboard/migrations/0012_auto_20190207_1457.py [hunk 1]
```diff
@@ -28,7 +34,7 @@ class Migration(migrations.Migration):
         ),
         migrations.AddField(
             model_name='profile',
-            name='job_location',
-            field=django.contrib.postgres.fields.jsonb.JSONField(default=dict),
+            name='resume',
+            field=models.FileField(blank=True, help_text='The avatar SVG.', null=True, upload_to=avatar.utils.get_upload_filename),
```
Comment (octavioamu) [inline app/dashboard/migrations/0012_auto_20190207_1457.py]: Just a minor thing, the help text is "the avatar svg"

File: app/assets/v2/css/tag.css [hunk 2]
```diff
@@ -90,3 +90,26 @@
   background-color: #E7F0FA;
   color: #6587AE;
 }
+
+
```
Comment (thelostone-mc) [inline app/assets/v2/css/tag.css]: removeeee :P

File: app/assets/v2/js/jobs.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    data: formData,
+    processData: false,
+    dataType: 'json',
+    contentType: false
+  };
+
+  $.ajax(profile).done(function(response) {
+    _alert(response.message, 'info');
+  }).fail(function(error) {
+    _alert(error, 'error');
```
Comment (thelostone-mc) [inline app/assets/v2/js/jobs.js]: Should we make it a readable error or is it okay ?

File: app/assets/v2/js/jobs.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  });
+};
+
+// let jobLocations = [];
```
Comment (thelostone-mc) [inline app/assets/v2/js/jobs.js]: we can remove this

File: app/assets/v2/js/jobs.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+let autocomplete;
+
+function initPlacecomplete() {
+  let input = document.getElementById('jobLocation');
+
+  var options = {
```
Comment (thelostone-mc) [inline app/assets/v2/js/jobs.js]: let?

File: app/assets/v2/js/jobs.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+});
+
+$('#jobSalary').on('change', function() {
+  let currentValue = $(this).val();
+
+  $(this).val(Number(currentValue.replace(',', '')).toLocaleString());
+});
+
+
+const checkFileSize = (elem, max_img_size) => {
```
Comment (thelostone-mc) [inline app/assets/v2/js/jobs.js]: I believe we have this already in grants, could we reuse that ?

File: app/dashboard/templates/ftux/job.html [hunk 7]
```diff
@@ -16,27 +16,109 @@
...
+  //   let params = []
+  //   if (libs && libs.length) {
+  //     params.push('libraries=' + libs.join(','))
+  //   }
+  //   const scriptElement = document.createElement('script')
+  //   scriptElement.async = true;
+  //   scriptElement.defer = true;
+  //   scriptElement.src =`https://maps.googleapis.com/maps/api/js?key=${API_KEY}&${params.join('&')}&callback=${CALLBACK_NAME}`
+  //   document.body.appendChild(scriptElement)
+  // }
```
Comment (thelostone-mc) [inline app/dashboard/templates/ftux/job.html]: purge?

File: app/assets/v2/js/jobs.js (hunk 6 above)
Comment (octavioamu) [reply app/assets/v2/js/jobs.js]: Is grants loading shared.js also? or just grants/shared.js?  I have seen a lot of duplicated function in grants/shared.js

**Pull request merged**